test/test_jit.py

```diff
@@ -1549,6 +1549,26 @@ def test_trace_size(self):
     def test_trace_size_with_grad(self):
         self.do_trace_size(True)
 
+    def do_trace_arange(self, requires_grad):
+        def fn(x):
+            return torch.arange(x.shape[0])
```

Comment:
maybe add test for tracing `arange` with a scalar input as well, and another test where both `start` and `end` are specified ?